creator-node/src/index.ts

```diff
@@ -100,6 +101,19 @@ const startApp = async () => {
     )
   }
 
+  try {
+    const solDelegateKeypair = Keypair.fromSeed(privateKeyBuffer)
+    const solDelegatePrivateKey = solDelegateKeypair.secretKey
+    config.set(
+      'solDelegatePrivateKeyBase64',
+      Buffer.from(solDelegatePrivateKey).toString('base64')
```

Comment:
why encode this as a base64 instead of storing Uint8Array?

Reply:
@hareeshnagaraj I was originally just setting the config option to a Uint8Array but it was harder to write the test without using an explicit encoding. my reasoning is also that it's technically a config option and convict doesn't have 'uint8array' as a type afaik. so if we want to be able to set it in the future then it should be in a format that convict supports (I'm also not sure how you'd set a Uint8Array env var)